pkg/ccl/backupccl/backup_processor.go

```diff
@@ -636,12 +636,13 @@ func runBackupProcessor(
 								resumeTS = resp.Files[fileCount-1].EndKeyTS
 							}
 							resumeSpan = spanAndTime{
-								span:       *resp.ResumeSpan,
-								firstKeyTS: resumeTS,
-								start:      span.start,
-								end:        span.end,
-								attempts:   span.attempts,
-								lastTried:  span.lastTried,
+								span:         *resp.ResumeSpan,
+								firstKeyTS:   resumeTS,
+								start:        span.start,
+								end:          span.end,
+								attempts:     span.attempts,
+								lastTried:    span.lastTried,
```

Comment:
Should we reconsider our pattern her. Perhaps we should copy the whole struct and then reset the fields we want to reset.

Reply:
perhaps in a follow up pr that doesn't get backported?

Reply:
done.